api/src/test/resources/io/strimzi/api/kafka/model/040-Crd-kafka-crdApi-v1.yaml

```diff
@@ -275,6 +275,15 @@ spec:
                               type: boolean
                               description: Enable or disable ECDSA support by installing
                                 BouncyCastle crypto provider. Default value is `false`.
+                            enableOauthBearer:
+                              type: boolean
+                              description: Enable or disable OAuth authentication
+                                over SASL_OAUTHBEARER.Default value is `true`.
```

Comment:
```suggestion
                                over SASL_OAUTHBEARER. Default value is `true`.
```

Reply:
Will have to fix this in the annotation in the CR class since that's the source from where this is generated as part of the build.